README.md

```diff
@@ -135,6 +136,7 @@ Yazılım geliştirme ile ilgili özenle seçilmiş Türkçe kaynaklar listesi.
 - [plepa.com](http://plepa.com/)
 - [cppdili](https://cppdili.blogspot.com.tr/)
 - [Qt 4 ile C++ GUI Programlama](https://ufukuzun.wordpress.com/yayinlarim/qt-4-ile-c-gui-programlama/) - [Ufuk Uzun](https://ufukuzun.wordpress.com/)
+- [C++ - learnXinYminutes](https://learnxinyminutes.com/docs/tr-tr/c++-tr/)
```

Comment:
Bu repository ile doğrudan alakalı değil; fakat şurada bir satır kaçmış gibi duruyor: https://github.com/adambard/learnxinyminutes-docs/blame/master/tr-tr/c++-tr.html.markdown#L35
Bir ara vakit olunca düzenlerseniz harika olur ;)

Reply:
Dikkatiniz için çok teşekkür ederim, onu ve diğer birkaç küçük hatayı düzelttim, ama ne zaman merge edilir bilmiyorum :))

Reply:
Teşekkürler ^_^